udata/core/dataset/api_fields.py

```diff
@@ -245,6 +253,8 @@
     'quality': fields.Raw(description='The dataset quality', readonly=True),
     'last_update': fields.ISODateTime(
         description='The resources last modification date', required=True),
+    'schema': fields.Nested(
```

Comment:
We may want to update `DEFAULT_MASK` on dataset fields to add schema? Not totally sure if we don't want to confuse unaware users